bark_ml/library_wrappers/lib_tf2rl/generate_expert_trajectories.py

```diff
@@ -280,6 +280,7 @@ def simulate_scenario(param_server: ParameterServer, sim_time_step: float, rende
 
     param_server["ML"]["StateObserver"]["MaxNumAgents"] = 3
 
+    # TODO Replace these with the C++ NearestObserver
     observer = NearestAgentsObserver(param_server)
```

Comment:
We should try replacing the `NearestAgentsObserver` (Python) with the `NearestObserver` (C++) as this is also used in the Merging Blueprint. I have already seen some flaws when using the python observer when training a SAC agent, so I think we have to use the c++ one.
This will most likely give seg fault errors in the bark project. We need to find out why.